fdbserver/worker.actor.cpp

```diff
@@ -723,56 +723,153 @@ TEST_CASE("/fdbserver/worker/addressInDbAndPrimaryDc") {
 
 } // namespace
 
+// Returns true if `address` is used in the db (indicated by `dbInfo`) transaction system and in the db's remote DC.
+bool addressInDbAndRemoteDc(const NetworkAddress& address, Reference<AsyncVar<ServerDBInfo> const> dbInfo) {
```

Comment:
remoteTransactionSystemContainsDegradedServers() can easily call addressInDbAndRemoteDc() to eliminate code duplication.

Reply:
I thought about this, but I'm not sure where to put it. There doesn't seem to be a `util` or `helper` class where we can put common helpers, at least I'm not aware of. Any suggestions?

Reply:
worker.actor.cpp is fine for me. Just add the declaration in WorkerInterface.actor.h then remoteTransactionSystemContainsDegradedServers() should be able to call it?

Reply:
Great suggestion!